bamboo/integration-tests.sh

```diff
@@ -4,4 +4,11 @@ set -e
 . ./bamboo/abort-if-skip-integration-tests.sh
 . ./bamboo/set-bamboo-env-variables.sh
 
-(cd example && npm install && npm test)
\ No newline at end of file
+if [[ $USE_NPM_PACKAGES == true ]]; then
```

Comment:
This value is set to allow publish job (and possibly others) to utilize npm to bring in packages instead of build them locally via bootstrap/build